src/Search_Replace_Command.php

```diff
@@ -258,7 +258,7 @@ public function __invoke( $args, $assoc_args ) {
 			} else {
 				$success_message = "Made $total replacements.";
 				if ( $total && 'Default' !== WP_CLI\Utils\wp_get_cache_type() ) {
-					$success_message .= ' Please remember to flush your persistent object cache with `wp cache flush`.';
+					$success_message .= ' Please remember to flush your persistent object cache with `wp cache flush`. If you see a "Site not found" error after replacing a domain, try flushing cache against the old domain (which may be the cached lookup value).';
```

Comment:
The "If you see..." clarification should only be included when `is_multisite()` is true, because it's not applicable for single site.